CLAUDE.md

```
@@ -20,13 +20,20 @@ go install github.com/minio/hperf/cmd/hperf@latest
 
 ### Run tests
 ```bash
-go test ./...
+go test -race ./...
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)